index.js

```diff
@@ -1,35 +1,46 @@
 "use strict";
 
-const d           = require("d")
-    , lazy        = require("d/lazy")
-    , BbPromise   = require("bluebird")
-    , Ec2         = require("aws-sdk/clients/ec2")
-    , { inspect } = require("util");
+const d = require("d")
+	, lazy = require("d/lazy")
+	, BbPromise = require("bluebird")
+	, { inspect } = require("util");
 
 const noopPromise = Promise.resolve();
 
 class ServerlessPluginVpcEniCleanup {
 	constructor(serverless) {
 		this.serverless = serverless;
+		this.provider = this.serverless.getProvider(this.serverless.service.provider.name);
 		this.hooks = {
 			"before:remove:remove": this.cleanup.bind(this),
 			"after:remove:remove": () => this.isDisabled = true
 		};
 	}
+
+	describeNetworkInterfaces(args) {
+		return this.provider.request("EC2", "describeNetworkInterfaces", args);
+	}
+
+	detachNetworkInterface(args) {
+		return this.provider.request("EC2", "detachNetworkInterface", args);
+	}
+
+	deleteNetworkInterface(args) {
+		return this.provider.request("EC2", "deleteNetworkInterface", args);
+	}
```

Comment:
Let's not introduce those methods, but instead, let's use `this.provider.request` directly

Reply:
done